src/SIL.LCModel/Infrastructure/Impl/SharedXMLBackendProvider.cs

```diff
@@ -262,7 +262,7 @@ private void CreateSharedMemory(bool createdNew)
 
 		private MemoryMappedFile CreateOrOpen(string name, long capacity, bool createdNew)
 		{
-			if (MiscUtils.IsMono)
+			if (MiscUtils.IsMono || !MiscUtils.IsWindows)
```

Comment:
Based on further conversation with @jasonleenaylor and @ermshiperete we will leave the Windows implementation alone and stick with a simple check for "isUnix":
```suggestion
			if (SIL.PlatformUtilities.Platform.IsUnix)
```